src/pages/[...slug].tsx

```diff
@@ -20,26 +16,21 @@ export default function NodePage({ node }) {
 
 export async function getStaticPaths(context) {
     return {
-        paths: await getPathsFromContext(['node--q_a'], context),
+        paths: await drupalClient.getPathsFromContext(['node--q_a'], context),
         fallback: false,
     };
 }
 
 export async function getStaticProps(context) {
 
-    const type = await getResourceTypeFromContext(context);
-    if (!type) {
-        return {
-            notFound: true,
-        };
-    }
+    const type = 'node--q_a';
```

Comment:
In the other cases, the previously available methods are now part of `client`, but there was no replacement for `getResourceTypeFromContext` that I could see.